content/docs/how-tos/mirroring-to-quay.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Mirror an image to an external registry"
+date: 2020-10-05T11:14:39-04:00
+draft: false
+---
+
+
+## Requirements
+
+In order to mirror an image to quay, that image must be have a [promotion
+configuration.](/docs/architecture/ci-operator/#publishing-container-images)
+
+If the image is promoted into a namespace for which no other image mirroring is set up yet, some RBAC needs to
+be configured:
+
+* In [roles.yaml](https://github.com/openshift/release/blob/master/core-services/various_rbac/admin_ci-operator_roles.yaml),
+  add role `image-tagger` and rolebinding `image-tagger-ci-operator` for your namespace.
+* The admin of the namespace should allow the SA in the mirroring job defined below to access the images with `oc image mirror`,
+	e.g.,
+  [admin_openshift-kni_list.yaml](https://github.com/openshift/release/blob/master/core-services/image-mirroring/admin_openshift-kni_list.yaml)
+  which makes the images open to the public.
+
+## Configuring mirroring for new images
+
+
+Simply submit a PR adding the image to the appropriate mapping file. You will
+need an approval of an owner of the image set. Each folder in the [image mirroring
+config](https://github.com/openshift/release/tree/master/core-services/image-mirroring) is an image set.
+
+## Configuring mirroring for new sets of images
+
+Submit a PR adding a new subdirectory
+[here](https://github.com/openshift/release/tree/master/core-services/image-mirroring), with at least a single mapping file
+and an `OWNERS` file (so that you can maintain your mappings). The mapping files
+should follow the `mapping_$name$anything` naming convention to avoid conflicts
+when put to a `ConfigMap`. Submit a PR which contains the new mappings, and after merging,
+we can expect to see them via
+
+{{< highlight bash >}}
+> oc get cm -n ci image-mirror-mappings -o yaml | yq -r '.data | keys[]'
```

Comment:
nit: use `extract` and always long-form flags in documentation

```
$ oc --context api.ci --namespace ci extract configmap/image-mirror-mappings
```

Also -- do we expect users to be doing this? If no, let's omit